core/dive.cpp

```diff
@@ -2399,7 +2401,7 @@ bool dive::cache_is_valid() const
 pressure_t dive::get_surface_pressure() const
 {
 	return surface_pressure.mbar > 0 ? surface_pressure
-					 : pressure_t { SURFACE_PRESSURE };
+					 : pressure_t { .mbar = SURFACE_PRESSURE };
```

Comment:
While we are playing with unit types, should we replace / augment `SURFACE_PRESSURE` with something that allows us to flex our new unit system when using it?